scripts/make_mavd_index.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+from natsort import natsorted, ns
+import json
+import os
+import glob
+from soundata.validate import md5
+
+INDEX_PATH = "../soundata/datasets/indexes/mavd_index.json"
+
+
+def make_index(data_path):
```

Comment:
@mrocamora @pzinemanas @justinsalamon this dataset has video, do we want to support it as well?

I personally think is a good idea since most audio-visual datasets are mostly used by the computer vision community, but I think we're missing out on a lot of potential there.

That said, if incorporating video support is complicated (e.g. which library to use) maybe let's get this loader done with audio only and add the video in a further PR?

Thoughts?

Reply:
I agree we should support video - AudioSet is another good example of a popular audio dataset that is made of video clips.

@magdalenafuentes am I right in that the only thing we need to figure out is how to load the video file? I guess we'll want to support both `load_audio` and `load_video` so the user can choose whether to load the audio or the visual frames?

Reply:
For `openl3` we used `moviepy`, which I believe is the go-to package for video manipulation in python:
https://zulko.github.io/moviepy/

Reply:
Yes, that's pretty much what we have to decide. I don't know `moviepy`, but looks good to me. @justinsalamon what do the CV people from your team recommend? Is it `moviepy`?

Reply:
I think so but let me confirm

Reply:
OK so there are a few popular ones, `moviepy`, `cv2` (OpenCV2), `decord`... but it seems like the most popular one right now is still `moviepy`, so I think we should go with this one. If users ask for supporting other video loaders we can address that down the road.